openscm/adapters/dice.py

```diff
@@ -0,0 +1,388 @@
+            ParameterType.POINT_TIMESERIES,
+            InterpolationType.LINEAR,
+            ExtrapolationType.LINEAR,
+        )
+
+        # Carbon concentration increase in shallow oceans (rel. to 1750)
+        self._views.M_u = self._output.get_writable_timeseries_view(
+            ("Concentration", "Ocean", "shallow"),
+            ("World",),
+            "GtC",
+            time_points,
+            ParameterType.POINT_TIMESERIES,
+            InterpolationType.LINEAR,
+            ExtrapolationType.LINEAR,
+        )
+
+        # Increase temperature of atmosphere (rel. to 1900)
+        self._views.T_atm = self._output.get_writable_timeseries_view(
+            ("Temperature Increase", "Atmosphere"),
+            ("World",),
+            "degC",
+            time_points,
+            ParameterType.POINT_TIMESERIES,
+            InterpolationType.LINEAR,
+            ExtrapolationType.LINEAR,
+        )
+
+        # Increase in temperature of lower oceans (rel. to 1900)
+        self._views.T_ocean = self._output.get_writable_timeseries_view(
+            ("Temperature Increase", "Ocean", "lower"),
+            ("World",),
+            "degC",
+            time_points,
+            ParameterType.POINT_TIMESERIES,
+            InterpolationType.LINEAR,
+            ExtrapolationType.LINEAR,
+        )
+
+        # Increase in radiative forcing (rel. to 1900)
+        self._views.force = self._output.get_writable_timeseries_view(
+            ("Radiative forcing",),
+            ("World",),
+            "W/m^2",
+            time_points,
+            ParameterType.POINT_TIMESERIES,
+            InterpolationType.LINEAR,
+            ExtrapolationType.LINEAR,
+        )
+
+    def _reset(self) -> None:
+        self._timestep = 0
+        for name in MODEL_PARAMETER_DEFAULTS:
+            setattr(self._values, name, getattr(self._views, name).get())
+
+        # Carbon cycle transition matrix
+        self._values.b11 = 1 - self._values.b12
+        self._values.b21 = (
+            self._values.b12 * self._values.M_atm_eq / self._values.M_u_eq
+        )
+        self._values.b22 = 1 - self._values.b21 - self._values.b23
+        self._values.b32 = self._values.b23 * self._values.M_u_eq / self._values.M_l_eq
+        self._values.b33 = 1 - self._values.b32
+
+        self._values.E = self._views.E.get()
+
+        self._values.M_atm = np.empty(self._timestep_count)
+        self._values.M_atm[0] = self._values.M_atm0
+        self._values.M_l = np.empty(self._timestep_count)
+        self._values.M_l[0] = self._values.M_l0
+        self._values.M_u = np.empty(self._timestep_count)
+        self._values.M_u[0] = self._values.M_u0
+        self._values.T_atm = np.empty(self._timestep_count)
+        self._values.T_atm[0] = self._values.T_atm0
+        self._values.T_ocean = np.empty(self._timestep_count)
+        self._values.T_ocean[0] = self._values.T_ocean0
+
+        self._values.force = np.empty(self._timestep_count)
```

Comment:
```suggestion
        self._values.rf = np.empty(self._timestep_count)
```

or `forcing` or `radiative_forcing`, `force` is not a commonly used abbreviation

Reply:
will keep the original names (though i also dont agree with most choices). will add acomment about that